generator/internal/codegen_utils.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef GOOGLE_CLOUD_CPP_GENERATOR_INTERNAL_CODEGEN_UTILS_H
+#define GOOGLE_CLOUD_CPP_GENERATOR_INTERNAL_CODEGEN_UTILS_H
+
+#include "absl/strings/string_view.h"
+#include <string>
+
+namespace google {
+namespace cloud {
+namespace generator_internal {
+
+/**
+ * Suffix for generated files to indicate what plugin generated them.
+ */
+std::string GeneratedFileSuffix();
+
+// Convenience functions for wrapping include headers with the correct
```

Comment:
Nit: this comment style doesn't match the other `/** */` comments in this file. Unless the difference is important, we probably want to be consistent.

Reply:
Done.